src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
-body {
+@import "variables";
+@import "mixins";
+
+@import "blocks/page";
+@import "blocks/container";
+@import "blocks/menu-container";
+@import "blocks/logo";
+@import "blocks/icon";
+@import "blocks/top-actions";
+@import "blocks/header";
+@import "blocks/menu";
+@import "blocks/grid";
+
+@import "blocks/why";
+@import "blocks/category";
+@import "blocks/button";
+@import "blocks/recommended";
+@import "blocks/contacts";
+@import "blocks/form-field";
+
+@import "blocks/footer";
+
+h2 {
```

Comment:
Consider adding this styling to your page styles file or to the basic style file where you would reset unnecessary browser styling.